fluent-plugin-google-cloud.gemspec

```diff
@@ -22,7 +22,7 @@ eos
   gem.add_runtime_dependency 'fluentd', '~> 0.10'
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
   gem.add_runtime_dependency 'google-api-client', '~> 0.9.0'
-  gem.add_runtime_dependency 'google-cloud-logging', '~> 0.23.2'
+  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.2.1'
```

Comment:
OK. This actually can work with exactly version 0.24.1, which is slightly older. I changed the gemspec to depend on that exact version.